src/components/About/Team/index.js

```diff
@@ -133,18 +131,81 @@ const Description = styled.div`
   padding: 1rem;
 `;
 
+const contributorsWebLink =
+  'https://api.github.com/repos/fossasia/susi.ai/contributors';
+const contributorsServerLink =
+  'https://api.github.com/repos/fossasia/susi_server/contributors';
 export default class Team extends Component {
   static propTypes = {
     history: PropTypes.object,
     location: PropTypes.object,
   };
 
-  componentDidMount() {
+  state = {
+    contributors: [],
+  };
+
+  async componentDidMount() {
     // Adding title tag to page
     document.title =
       'Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
+
+    let contributors = [];
+    let response = await axios.get(contributorsWebLink);
```

Comment:
Please move all API calls to apis file. It internally uses `axios` to make API calls.